harvester/Dockerfile

```diff
@@ -63,8 +63,7 @@ RUN cat /usr/src/app/harvester/bin/tika-app-1.24.1.jar.sha512 | echo "$(</dev/st
 # Entrypoint sets our environment correctly
 ENTRYPOINT ["/usr/src/app/entrypoint.sh"]
 
-# The default command is to start a supervisord that runs: Celery, Celery Beat and a small UWSGI
-CMD ["supervisord", "-c", "/usr/src/app/harvester/supervisord/main.conf"]
+CMD ["uwsgi", "--ini", "/usr/src/app/uwsgi.ini", "--logger", "file:logfile=/tmp/harvest-uwsgi.log,maxsize=2000000"]
```

Comment:
Can you explain the logfile setting? Is that necessary voor firelens? Why can all other containers run without the log file? (a log file that we're never able to see on AWS afaik)

Reply:
I just copied this from the existing supervisord config. The harvester still uses cloudwatch logs, so I think that's the reason 😉

Reply:
Well. In supervisord we need to silence the UWSGI. With a tasks setup I expect we won't have to. So that logger part should be removed. Not sure how relevant it is as we're not merging this, right? Just a FYI :)

Reply:
Uh, the idea is to merge this 😄 . It removes the supervisord part. I didn't add nginx yet, because it didn't work somehow. So it isn't available over the internet. However this does exactly the same as the current situation without supervisord. We can access it through the bastion host.
I will remove the logging part next monday